config.go

```diff
@@ -440,9 +454,7 @@ type Config struct {
 	// we select. This means, the larger this number is, the longer we need to
 	// hold this lock. On the flip side, the smaller this number is, the slower
 	// the copy will likely be.
-	//
-	// Optional: defaults to 200
-	DataIterationBatchSize uint64
+	DataIterationBatchSize DataIterationBatchSize
```

Comment:
This is breaking config backward-compatibility, is this intended? We could also keep the default variable as is and have the map as a separate struct if we want to keep compability.

Reply:
Agreed. we should not break backward compatibility right now. We can have an override config:

```
DataIterationBatchSize uint64
DataIterationBatchSizePerTableOverride ....
```

Reply:
not intended, will use a separate `DataIterationBatchSizePerTableOverride` to not break backwards compatibility